python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -206,6 +206,18 @@
           )
         end
       end
+
+      context "when the marker is malformed" do
+        before do
+          allow(parser).to receive(:python_raw_version).and_return("3.13.3")
+        end
+
+        let(:requirements_fixture_name) { "malformed_markers.txt" }
+
+        it "then the dependency version should be 1.0.4" do
```

Comment:
The test description doesn't match the `empty`?

Reply:
Copy and paste did me, there